src/components/Dropdown/computeDropdownStyles.ts

```diff
@@ -0,0 +1,32 @@
+import { ElementPositionAndSize } from '../../hooks/useElementPositionAndSize';
```

Comment:
This should be `import type` instead of just `import`

Reply:
Thank you. Updated